site/troubleshooting.md

```diff
@@ -162,8 +166,8 @@ The columns are as follows:
       heartbeat
     * `established` - TCP connection and corresponding UDP path are up
  * Info - the failure reason for failed and retrying connections, or
-   the data transport method, remote peer name and nickname for
-   pending and established connections
+   the data transport method, security level, remote peer name and
+   nickname for pending and established connections
```

Comment:
Ack on the ordering - good spot - will update. Some things about the field itself:

* `plaintext` and `encrypted` were chosen as the two values because they are the same length
* I'd like the output to contain a positive indication that things are going in the plain, and not rely on e.g. someone noticing the absence of an '(encrypted) affix
* It's deliberately separate from the overlay type so we don't have to tinker with the status output when fastdp supports encryption